CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+defaang lives through contributions from the community. In contributing to this repository, please first discuss the change you wish to make via an issue, discord, or any other method with the maintainers of this repository before making a change.
+
+**Table of Contents**
+
+- [Contributing](#contributing)
+  - [Where to Get Help](#where-to-get-help)
+    - [All Communication Channels](#all-communication-channels)
+  - [How to Help](#how-to-help)
+    - [Our Stack (with linked documentation)](#our-stack)
+    - [Issue Labels](#issue-labels)
+    - [Semantic Commits](#semantic-commits)
+    - [Commit Example](#commit-example)
+  - [Remarks](#remarks)
+
+## Where to Get Help
+
+An active community forms the backbone of every major Open Source project. The community is present on various platforms on the internet, where we actively help each other out – not only with regard to using the application. GitHub is the most tech-focused part of the community, albeit [discussing here](https://github.com/ykdojo/defaang/discussions) is the way to go for more theoretical discussions. Furthermore, we have a [Discord-server](https://discord.gg/aJp6Fypb) where we discuss building the app and help each other out.
+
+### All Communication Channels
+
+- [Discord](https://discord.gg/aJp6Fypb)
+- [GitHub](https://github.com/ykdojo/defaang/discussions)
+
+## How to Help
+
+If you have some experience with developing, we would gladly welcome your help in fixing issues and adding features. Even if you just start: There are tons of smaller things and chores to do, so by cleaning up code here and there you can improve your skills and also help the project. Even if it's a minor fix, it is still a fix!
```

Comment:
I really don't like the word chores here - I really don't think we have any *chores* to do? I think every task is interesting in a different way